pkg/component/watch.go

```diff
@@ -58,7 +62,7 @@ func isRegExpMatch(strToMatch string, regExps []string) (bool, error) {
 // and its subdirectories.  If a non-directory is specified, this call is a no-op.
 // Files matching glob pattern defined in ignores will be ignored.
 // Taken from https://github.com/openshift/origin/blob/85eb37b34f0657631592356d020cef5a58470f8e/pkg/util/fsnotification/fsnotification.go
-func addRecursiveWatch(watcher *fsnotify.Watcher, path string, ignores []string) error {
+func addRecursiveWatch(watcher *fsnotify.Watcher, directory, path string, ignores []string) error {
```

Comment:
how are  `directory` and `path` different?

Reply:
can we use a little bit more descriptive name for `directory` and `path` to make it a little bit more apparent what is what?